samples/isilon_v270_k8s_124.yaml

```diff
@@ -5,15 +5,15 @@ metadata:
   namespace: test-isilon
 spec:
   driver:
-    # Config version for CSI PowerScale v2.4.0 driver
-    configVersion: v2.4.0
+    # Config version for CSI PowerScale v2.6.0 driver
+    configVersion: v2.6.0
```

Comment:
Please change the config version and the corresponding comment to 2.7.0 . Same comment for all the K8s samples